page-factory-core/src/main/java/ru/sbtqa/tag/pagefactory/tasks/TaskHandler.java

```diff
@@ -0,0 +1,20 @@
+package ru.sbtqa.tag.pagefactory.tasks;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskHandler {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)